Modules/+Sources/TunableLaser_invisible.m

```diff
@@ -58,6 +62,8 @@ function trackFrequency(obj,varargin)
             freqH = plot(ax(1),NaN,NaN,'r-o','DisplayName','Current Frequency');
             dfreqH = plot(ax(2),NaN,NaN,'r-o');
             legend(ax(1),'show');
+            
+            freq = obj.getFrequency;
             n = 0;
             while obj.tuning
                 freq = obj.getFrequency;
```

Comment:
is there a change obj.tuning will be false, such that line 66 will not be redundant with line 69?

Reply:
obj.tuning is set by obj.getFrequency. getFrequency does many comms with the laser.